tests/tests/test-crowdloan.ts

```diff
@@ -0,0 +1,364 @@
+import { Keyring } from "@polkadot/api";
+import { KeyringPair } from "@polkadot/keyring/types";
+import { expect } from "chai";
+import Web3 from "web3";
+import { Account } from "web3-core";
+
+import {
+  GENESIS_ACCOUNT,
+  GENESIS_ACCOUNT_BALANCE,
+  GENESIS_ACCOUNT_PRIVATE_KEY,
+  GLMR,
+  ALITH_PRIV_KEY,
+  ALITH,
+} from "../util/constants";
+import { describeDevMoonbeam } from "../util/setup-dev-tests";
+
+describeDevMoonbeam("Crowdloan", (context) => {
+  let genesisAccount: KeyringPair;
+  const relayChainAddress: string = "couldBeAnyString";
+  before("Setup genesis account for substrate", async () => {
+    const keyring = new Keyring({ type: "ethereum" });
+    genesisAccount = await keyring.addFromUri(GENESIS_ACCOUNT_PRIVATE_KEY, null, "ethereum");
+  });
+  it("should be able to register the genesis account for reward", async function () {});
+});
+describeDevMoonbeam("Crowdloan", (context) => {
+  let genesisAccount: KeyringPair;
+  const relayChainAddress: string = "couldBeAnyString";
+  before("Setup genesis account for substrate", async () => {
+    const keyring = new Keyring({ type: "ethereum" });
+    genesisAccount = await keyring.addFromUri(GENESIS_ACCOUNT_PRIVATE_KEY, null, "ethereum");
```

Comment:
Why do you need the account? you can simply provide the private key to the signAndSend